File: Sportivo_webAPI/Models/Company.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Sportivo_webAPI.Models
{
    public class Company
    {
        public int CompanyId { get; set; }
        [Required]
        public string CompanyName { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public string PhoneNumber { get; set; }
        public string Description { get; set; }
        public ICollection<Court> Courts { get; set; }
        public int Opens { get; set; }
        public int Closes { get; set; }
    }
}

File: Sportivo/app/src/main/java/com/example/sportivo/Models/Company.java
package com.example.sportivo.Models;

import com.example.sportivo.Models.Court;

import java.util.ArrayList;

public class Company {

    private String companyName;
    private int companyId;
    private ArrayList<Court> courts;
    private int opens;
    private int closes;

    public Company(int ID, String name){

        this.companyName=name;
        this.companyId=ID;
    }
    public String getName(){
        return this.companyName;
    }
    public int getId(){
        return this.companyId;
    }
    public int getOpens() { return this.opens; }
    public int getCloses() { return this.closes; }
    public int getCompanyId() { return this.companyId; }

    public ArrayList<Court> getCourts() {
        return courts;
    }
}

